views/beneficial-owner-statements.html

```diff
@@ -43,15 +35,15 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
           items: [
             {
               value: "ALL_IDENTIFIED_ALL_DETAILS",
-              text: "All beneficial owners have been identified and all required information can be provided",
+              text: "All beneficial owners have been identified and I can provide all the required information",
```

Comment:
We need to make changes on check your answers page [here](https://github.com/companieshouse/overseas-entities-web/blob/main/views/includes/check-your-answers/beneficial-owner-statements.html) as well.

Could you please add a few text check on related unit tests, thanks Sam

Reply:
Ok, I can do that